projects/social/social.py

```diff
@@ -45,8 +47,30 @@ def populateGraph(self, numUsers, avgFriendships):
         self.users = {}
         self.friendships = {}
         # !!!! IMPLEMENT ME
+        if numUsers <= avgFriendships:
```

Comment:
I may be reading this wrong but it looks like while you're creating friendships, you aren't building out the graph. 

This is the way I solved this problem:

<img width="476" alt="Screen Shot 2019-06-20 at 11 09 18 AM" src="https://user-images.githubusercontent.com/33299372/59867599-deb8e400-934b-11e9-828b-af49bbcd142c.png">